app-android-xamarin/Moodis/Moodis/Feature/Menu/MenuActivity.cs

```diff
@@ -0,0 +1,155 @@
+            {
+                label.Text = "loading";
+            }
+            /* COMENTED UNTIL WE FIGURE OUT A WAY TO STORE ENVIROMENTAL VARIABLES
+            try
+            {
+                await ActivityMenuViewModel.GetFaceEmotionsAsync();
+            }
+            catch (System.Net.Http.HttpRequestException e)
+            {
+                Log.Debug(TAG,e.Message);
+                Toast.MakeText(this, WarningInRequest, ToastLength.Short).Show();
+                JavaSystem.Exit(0);
+            }
+            */
+            if (ActivityMenuViewModel.currentImage.emotions != null)
+            {
+                var counter = 0;
+                foreach (var label in emotionLabels)
+                {
+                    label.Text = ActivityMenuViewModel.currentImage.emotions[counter].name + " : "
+                        + ActivityMenuViewModel.currentImage.emotions[counter].confidence.ToString(FormatDouble);
+                    counter++;
+                }
+                ActivityMenuViewModel.UserAddImage();
+            }
+            else
+            {
+                Toast.MakeText(this, WarningFaceDetection, ToastLength.Short).Show();
+            }
+        }
+
+        private Bitmap RotateImage(Bitmap image)
+        {
+            Matrix matrix = new Matrix();
+            matrix.PostRotate(-90);
+            return Bitmap.CreateBitmap(image, 0, 0, image.Width, image.Height, matrix, true);
+        }
+        public override void OnBackPressed()
+        {
+            //Finish(); TODO figure out what to do when back pressed while in menu (maybe logout user ?)
+        }
+
+        private void InitButtonsAndInputs()
+        {
+            var bntToCalendar = FindViewById(Resource.Id.goToCalendar);
+            var btnPlayMusic = FindViewById(Resource.Id.playMusic);
+            var btnStopMusic = FindViewById(Resource.Id.StopMusic);
+            var btnGroups = FindViewById(Resource.Id.groups);
+            var btnTakePicture = FindViewById(Resource.Id.goToCamera);
+            bntToCalendar.Click += (sender, e) => {
+                BtnToCalendar_Click();
+            };
+            btnPlayMusic.Click += (sender, e) => {
+                BtnPlayMusic_Click();
+            };
+            btnStopMusic.Click += (sender, e) => {
+                BtnStopMusic_Click();
+            };
+            btnGroups.Click += (sender, e) => {
+                BtnGroups_Click();
+            };
+            btnTakePicture.Click += (sender, e) => {
+                BtnTakePicture_Click();
+            };
+        }
+
+        private void DeleteImage(string path)
+        {
+            if (File.Exists(path))
+            {
+                File.Delete(path);
+            }
+        }
+
+        private void BtnTakePicture_Click()
+        {
+            var cameraActivity = new Intent(this, typeof(CameraActivity));
+            StartActivity(cameraActivity);
+            Finish();
```

Comment:
Don't finish this activity, let it stay in the back stack, so when the cameraActivity finishes, then there would be no need to start menu activity again. In this case, the photo could be updated in OnResume method.